nion/swift/ExportDialog.py

```diff
@@ -246,66 +250,176 @@ def cancel(self) -> bool:
         return True
 
 
-class ExportSVGHandler:
-    def __init__(self, display_size: Geometry.IntSize) -> None:
-        self.width_model = Model.PropertyModel(display_size.width)
-        self.height_model = Model.PropertyModel(display_size.height)
+class UnitType(enum.Enum):
+    PIXELS = 0
+    INCHES = 1
+    CENTIMETERS = 2
+
+
+ConversionUnits = {
+    UnitType.PIXELS: 1,
+    UnitType.CENTIMETERS: 37,
+    UnitType.INCHES: 96
+}
 
-        self.int_converter = Converter.IntegerToStringConverter()
 
+class ExportSizeModel(Observable.Observable):
+    def __init__(self, display_item: DisplayItem.DisplayItem) -> None:
+        super().__init__()
+        display_size = self.__determine_display_size(display_item)
```

Comment:
Please add units to the function name - it will help prevent errors like the one that is trivially occurring.

`__determine_display_size` -> `__calculate_display_size_in_pixels`